flocker/node/agents/functional/test_cinder.py

```diff
@@ -224,7 +224,9 @@ def test_verify_false(self):
         config['peer_verify'] = False
         session = get_keystone_session(**config)
         region = get_openstack_region_for_test()
-        cinder_client = get_cinder_v1_client(session, region)
+        cinder_client = get_cinder_client(
+            session, region, version=config["cinder_api_version"]
```

Comment:
This is broken. 
Should re-use the same driver loading function as the IBlockDeviceAPI tests.